lib/api/actions/functions.php

```diff
@@ -410,119 +410,189 @@ function beans_reset_action( $id ) {
 }
 
 /**
- * Get action.
+ * Get the action's configuration for the given ID and status. Returns `false` if the action is not registered with
+ * Beans.
  *
+ * @since  1.0.0
+ * @since  1.5.0 Made WPCS compliant.
```

Comment:
I don't think it is necessary to add a note for `Made WPCS compliant.` which will become pretty redundant throughout the code base. It isn't really justified since no functionality was changed.

PS: this comment applies throughout the file.

Reply:
Here's from the [WordPress Handbook](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#since-section-changelogs):

>PHPDoc supports multiple @since versions in DocBlocks for this explicit reason. When adding changelog entries to the @since block, a version should be cited, and a description should be added in sentence case and form and end with a period:
```
* @since 3.0.0
* @since 3.8.0 Added the `post__in` argument.
* @since 4.1.0 The `$force` parameter is now optional.
```

Let's review then our revision control to ensure we do this right across the board.  

Each function will at a minimum receive these revisions:

1. Refactoring to make it WPCS complaint
2. Comments/documentation improvements

Now we have a decision to make.  How would you like to handle the `@since`?

I had originally upticked the version to `@since 1.5.0`, removing the `1.0.0`.  But then after reading the manual again, that's when I made the above change.

What do you think?  How should we capture the version and revisions for the `1.5.0` work?

Reply:
From my perspective only actually functionality changes justify to have a new since tag, not formatting changes. For instance, `_beans_set_action()` should have a new `@since 1.5.0 $action must be an array` since that is a new requirement.

Reply:
The latest commit has removed all of the `@since 1.5.0` from the private functions in this PR, except for the new `_beans_is_action_valid` function.